ifao-publications.csl

```diff
@@ -0,0 +1,335 @@
+    </group>
+  </macro>
+  <macro name="author-short">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name form="short" delimiter=", " et-al-min="4" et-al-use-first="1" font-variant="small-caps"/>
+          <et-al font-variant="normal" font-style="italic"/>
+        </names>
+      </if>
+      <else-if variable="editor">
+        <names variable="editor">
+          <name form="short" delimiter=", " et-al-min="4" et-al-use-first="1" font-variant="small-caps"/>
+          <label form="short" text-case="lowercase" font-variant="normal" prefix=" (" suffix=")"/>
+        </names>
+      </else-if>
+      <else>
+        <text variable="title-short" font-variant="small-caps"/>
+      </else>
+    </choose>
+    <text macro="year-date" prefix=" "/>
+  </macro>
+  <macro name="access">
+    <group delimiter=" ">
+      <text term="accessed"/>
+      <date variable="accessed" delimiter=" ">
+        <date-part name="day"/>
+        <date-part name="month"/>
+        <date-part name="year"/>
+      </date>
+    </group>
+  </macro>
+  <macro name="collection">
+    <group delimiter=" ">
+      <text variable="collection-title"/>
+      <text variable="collection-number"/>
+    </group>
+  </macro>
+  <macro name="title">
+    <text variable="title" text-case="title"/>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=", ">
+      <text variable="publisher-place"/>
+    </group>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <group delimiter=", ">
+      <date variable="original-date">
+        <date-part name="year"/>
+      </date>
+      <choose>
+        <if is-numeric="edition">
+          <group delimiter=" ">
+            <number variable="edition" form="ordinal"/>
+            <text term="edition" form="short" text-case="lowercase"/>
+          </group>
+        </if>
+        <else>
+          <text variable="edition"/>
+        </else>
+      </choose>
+    </group>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-year-suffix="true">
+    <sort>
+      <key variable="issued"/>
+    </sort>
+    <layout delimiter="&#160;; ">
+      <text macro="author-short"/>
+      <group prefix=", " font-variant="normal">
+        <label variable="locator" form="short" suffix=".&#160;"/>
+        <text variable="locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography>
+    <sort>
+      <key macro="author-short"/>
+      <key variable="issued"/>
+      <key variable="page-first"/>
+      <key variable="title"/>
+    </sort>
+    <layout suffix=".">
+      <group display="block">
+        <text macro="author-short"/>
+      </group>
+      <group display="left-margin">
```

Comment:
You have tested this? I'd have expected that just setting `display="block"` on this group (and no display on author short) would give you better results.

Reply:
It works perfectly on the test pane, less well on Word. I remember we had the same conversation about gallia.csl long ago (either here or on Zotero's forum). To make it work on Word, I need to add `suffix="&#9;"` after author-short. Just done.